src/main/resources/db/changelog/changes/07-create_shopping_carts_table.yaml

```
@@ -0,0 +1,28 @@
+databaseChangeLog:
+  - changeSet:
+      id: create_shopping_carts_table
+      author: DEz
```

```suggestion
      author: Volodymyr Ozemblovskyi
```
it's better to write full name of changeset's author